views/register.pug

```diff
@@ -74,7 +78,7 @@ block content
 
       p If you agree to these terms, please keep a copy of this form for your records.
 
-    div.form-group
+    //div.form-group
```

Comment:
This bit makes the tests fail, because we check that the checkbox is present. Let's remove this part.